modules/virt-troubleshooting-incorrect-policy-config.adoc

```diff
@@ -19,7 +19,7 @@ the available nmstate resources. You can then update the Policy with the correct
 +
 [source,yaml]
 ----
-apiVersion: nmstate.io/v1alpha1
+apiVersion: nmstate.io/v1beta1
```

Comment:
Same

Reply:
@phoracek indicated this was `nmstate.io/v1alpha1` above